docker/aeolus.yml

```diff
@@ -5,22 +5,9 @@ services:
         image: ghcr.io/ls1intum/aeolus/api:nightly
         pull_policy: always
         restart: unless-stopped
-        environment:
-            BAMBOO_GENERATOR_API_HOST: "http://aeolus-bamboo-api:8091"
         ports:
             - "8090:80"
         networks:
-            - aeolus
-            - artemis
-    aeolus-bamboo-api:
-        container_name: aeolus-bamboo-api
-        hostname: aeolus-bamboo-api
-        image: ghcr.io/ls1intum/aeolus/bamboo-generator:nightly
-        command: "--api"
-        pull_policy: always
-        restart: unless-stopped
-        networks:
```

Comment:
Is it on purpose that aeolus got removed as network?

Reply:
yes, with just the one container it's not required anymore.